doc/man7/EVP_SIGNATURE-ML-DSA.pod

```diff
@@ -70,7 +72,12 @@ to be set to 32 bytes of zeros. This value is ignored if "test-entropy" is set.
 
 The default value of 0 causes sign and verify operations to process a raw message.
 Setting this to 1 causes those operations to assume the input is the C<mu> value
-as defined in L<FIPS 204|https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.204.pdf>.
+from L<FIPS 204|https://csrc.nist.gov/pubs/fips/204/final> Algorithm 7 step 6 and
+Algorithm 8 step 7.
+
+Note that the message encoding steps from
+L<FIPS 204|https://csrc.nist.gov/pubs/fips/204/final> Algorithm 2 step 10 and
```

Comment:
I meant just saying that "message-encoding" is ignored :)

Reply:
Can you also change line 58... I think we should not say anything about pre hash

Reply:
Added.  I've removed the pre-hash text there but not removed it on lines 31-33 since I think it does deserve mention somewhere.